.github/workflows/main.yml

```
@@ -6,21 +6,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '2.6', '2.7' ]
-        rails: [ '5.0', '5.1', '5.2' ]
-        continue-on-error: [false]
-        include:
-          - ruby: '3.0'
-            rails: '5.0'
-            continue-on-error: true
-          - ruby: '3.0'
-            rails: '5.1'
-            continue-on-error: true
-          - ruby: '3.0'
-            rails: '5.2'
-            continue-on-error: true
+        ruby: [ '2.7', '3.0', '3.1', '3.2' ]
+        rails: [ '6.1', '7.0' ]
+        continue-on-error: [true]
     continue-on-error: ${{ matrix.continue-on-error }}
-    name: Test on Ruby ${{ matrix.ruby }}
+    name: Test on Ruby ${{ matrix.ruby }} x ${{ matrix.rails }}
```

(nit) Rails のバージョンであることが分かる名前のほうがよさそうです

```suggestion
    name: Test on Ruby ${{ matrix.ruby }} x Rails ${{ matrix.rails }}
```